cudamapper/src/main.cu

```diff
@@ -289,17 +289,33 @@ int main(int argc, char* argv[])
     claragenomics::logging::Init();
 
     const ApplicationParameteres parameters = read_input(argc, argv);
+    std::shared_ptr<claragenomics::io::FastaParser> query_parser;
 
-    std::shared_ptr<claragenomics::io::FastaParser> query_parser = claragenomics::io::create_kseq_fasta_parser(parameters.query_filepath, parameters.k + parameters.w - 1);
-
+    try
```

Comment:
While we should probably catch all exceptions in main, I would not do this for individual function calls.
For now I would rely on the C++ implementation to have a good handler for all uncaught exceptions which prints the error for us and ends the program.

Reply:
done.